app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -295,6 +296,10 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
             }, "externalApp")
         }
 
+        val cookieManager = CookieManager.getInstance()
+        cookieManager.setAcceptCookie(true)
+        CookieManager.getInstance().setAcceptThirdPartyCookies(webView, true)
```

Comment:
Why can't this just be `cookieManager.setAcceptThirdPartyCookies(webView, true)`?